formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "lead_organisations"
+  ],
+  "properties": {
+    "related_guides": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_format_provider_organisation": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "related_mainstream": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "additional_related_mainstream": {
```

Comment:
Fair enough, I forgot about the ordering.

I think I prefer the current implementation, however. It doesn't require the collating of two disparate properties to figure out which is the `main` and what's the `additional` content. And then you have to search an array for a matching ID instead of just directly calling a property; it's clunkier to use. Explicit property names document slightly better as well.

But I'll defer to you; if you're sure, then let's do the `details. related_mainstream_content ` version.